readme.md

```diff
@@ -257,3 +257,7 @@ Pull requests adding either experimental or mature implementations to these list
 - [graphql-java-kickstart/graphql-java-servlet](https://github.com/graphql-java-kickstart/graphql-java-servlet) (Java: [Maven](https://mvnrepository.com/artifact/com.graphql-java/graphql-java-servlet))
 - [ChilliCream/hotchocolate](https://github.com/ChilliCream/hotchocolate) (C#: [NuGet](https://www.nuget.org/packages/HotChocolate))
 - [async-graphql/async-graphql](https://github.com/async-graphql/async-graphql) (Rust: [Crates](https://crates.io/crates/async-graphql))
+
+### IDE/Tools
+
+- [Altair GraphQL](https://www.altairgraphql.dev/)
```

Comment:
Sorry for the late review! Thanks for all your hard work on Altair 🙏

The intention of this ***Implementations*** section is to:

- Show open source code examples of implementations of this spec.
- List key open source packages that can be used to implement this spec.

It's not attempting to link to the thousands? of products that generally make use of GraphQL file uploads.

In this section, "clients" are code that sends a request, and "servers" are code that receives a request. Altair is definitely a "client", because it doesn't receive an incoming multipart GraphQL request, it only sends them. As such, Altair is actually already listed in the "clients":

https://github.com/jaydenseric/graphql-multipart-request-spec/blob/460043baeb7f2f83ab55a11146946966df4cb222/readme.md?plain=1#L229